src/udev/udev-builtin-net_id.c

```diff
@@ -161,6 +167,55 @@ static struct udev_device *skip_virtio(struct udev_device *dev) {
         return parent;
 }
 
+static int get_virtfn_info(struct udev_device *dev, struct netnames *names, struct virtfn_info *vf_info) {
```

Comment:
we generally follow the rule that functions shouldn't clobber their return arguments on failure. Please follow this here too, i.e. initialized vf_info only on success, and leave it unmodified on failure. 

This doesn't really matter for the code flow here, but it makes this easier to maintain in the long run. 

And yes, not all our code follows this rule yet, but new code really should